test/aqua.jl

```diff
@@ -38,7 +39,7 @@ using Aqua
 
     @testset "Aqua tests (additional)" begin
         Aqua.test_undefined_exports(CloudMicrophysics)
-        Aqua.test_stale_deps(CloudMicrophysics)
+        Aqua.test_stale_deps(CloudMicrophysics; ignore = [:CUDA])
```

Comment:
Why do we need to ignore CUDA? It doesn't look like it's a dependency

Reply:
It gets added for GPU tests during the [pipeline](https://github.com/CliMA/CloudMicrophysics.jl/blob/main/.buildkite/pipeline.yml#L25)- I don't think it's necessary to add the package into CloudMicrophysics but I didn't want to mess with the pipeline.